spark/v3.4/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestConflictValidation.java

```diff
@@ -409,20 +414,21 @@ public void testOverwritePartitionNoSnapshotIdValidation() throws Exception {
 
     // Validating from null snapshot is equivalent to validating from beginning
     Dataset<Row> conflictingDf = spark.createDataFrame(records, SimpleRecord.class);
-    AssertHelpers.assertThrows(
-        "Conflicting deleted data files should throw exception",
-        ValidationException.class,
-        "Found conflicting files that can contain records matching partitions [id=1]",
-        () -> {
-          try {
-            conflictingDf
-                .writeTo(tableName)
-                .option(SparkWriteOptions.ISOLATION_LEVEL, IsolationLevel.SERIALIZABLE.toString())
-                .overwritePartitions();
-          } catch (NoSuchTableException e) {
-            throw new RuntimeException(e);
-          }
-        });
+    Assertions.assertThatThrownBy(
+            () -> {
+              try {
```

Comment:
we can remove all try-catch inside `assertThatThrownBy` blocks

Reply:
Thanks, already removed all~